features/conference/navigationHandler.native.js

```diff
@@ -0,0 +1,19 @@
+import { ScreenRegistry } from '../base/navigation';
+import { CONFERENCE_SCREEN } from './constants';
+
+/**
+ * Handler for conference screen route.
+ *
+ * @param {Store} store - Redux store.
+ * @param {Object} action - Action object.
+ * @param {Navigator} action.navigator - Navigator instance.
+ * @returns {void}
+ */
+export function navigate(store, action) {
+    let route = ScreenRegistry.getScreenByName(CONFERENCE_SCREEN);
+
+    // TODO: currently replace method doesn't support animation, but work
+    // towards adding it is done in
+    // https://github.com/facebook/react-native/issues/1981
+    action.navigator.replace(route);
```

Comment:
@tsareg, the change from the push method to the replace method is in its own (i.e. independently of the other modifications in this PR) the solution to the huge memory retention, right?

Initially, you started with the jumpTo method (as I remember you telling us in a meeting & I see in the first commit in this PR). Did you switch to the replace method later on in order to have exactly 1 mounted component/route/scene?